kotlin-jvm/build.gradle.kts

```diff
@@ -49,6 +58,7 @@ dependencies {
     implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:_")
     implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:_")
     implementation("org.kodein.di:kodein-di:_")
+    implementation ("com.squareup.wire:wire-runtime:3.4.0")
```

Comment:
Did you read the contributing guide? If so, please read it again, otherwise, please read it, there's some important considerations regarding adding dependencies.